src/i18n/locales/en.ts

```diff
@@ -1,395 +1,393 @@
-            close: 'Not now',
-            confirm: 'Of course!',
-        },
-        export_diagram_dialog: {
-            title: 'Export Diagram',
-            description: 'Choose the format for export:',
-            format_json: 'JSON',
-            cancel: 'Cancel',
-            export: 'Export',
-            error: {
-                title: 'Error exporting diagram',
-                description:
-                    'Something went wrong. Need help? chartdb.io@gmail.com',
-            },
+                'Something went wrong. Need help? chartdb.io@gmail.com',
         },
+    },
 
-        import_diagram_dialog: {
-            title: 'Import Diagram',
-            description: 'Paste the diagram JSON below:',
-            cancel: 'Cancel',
-            import: 'Import',
-            error: {
-                title: 'Error importing diagram',
-                description:
-                    'The diagram JSON is invalid. Please check the JSON and try again. Need help? chartdb.io@gmail.com',
-            },
-        },
-        relationship_type: {
-            one_to_one: 'One to One',
-            one_to_many: 'One to Many',
-            many_to_one: 'Many to One',
-            many_to_many: 'Many to Many',
+    import_diagram_dialog: {
+        title: 'Import Diagram',
+        description: 'Paste the diagram JSON below:',
+        cancel: 'Cancel',
+        import: 'Import',
+        error: {
+            title: 'Error importing diagram',
+            description:
+                'The diagram JSON is invalid. Please check the JSON and try again. Need help? chartdb.io@gmail.com',
         },
+    },
+    relationship_type: {
+        one_to_one: 'One to One',
+        one_to_many: 'One to Many',
+        many_to_one: 'Many to One',
+        many_to_many: 'Many to Many',
+    },
 
-        canvas_context_menu: {
-            new_table: 'New Table',
-            new_relationship: 'New Relationship',
-        },
+    canvas_context_menu: {
+        new_table: 'New Table',
+        new_relationship: 'New Relationship',
+    },
 
-        table_node_context_menu: {
-            edit_table: 'Edit Table',
-            duplicate_table: 'Duplicate Table',
-            delete_table: 'Delete Table',
-        },
+    table_node_context_menu: {
+        edit_table: 'Edit Table',
+        duplicate_table: 'Duplicate Table',
+        delete_table: 'Delete Table',
+    },
 
-        snap_to_grid_tooltip: 'Snap to Grid (Hold {{key}})',
+    snap_to_grid_tooltip: 'Snap to Grid (Hold {{key}})',
 
-        tool_tips: {
-            double_click_to_edit: 'Double-click to edit',
-        },
+    tool_tips: {
+        double_click_to_edit: 'Double-click to edit',
+    },
 
-        language_select: {
-            change_language: 'Language',
-        },
+    language_select: {
+        change_language: 'Language',
     },
-};
+} as const;
 
-export const enMetadata: LanguageMetadata = {
+export const enMetadata = {
```

Comment:
I tend to use keywords like as or satisfies (which I like much better) only when needed. Here for example, we can declare `enMetadata: LanguageMetadata` and it would serve the same purpose.
What do you think? Do you prefer satisfies keyword instead of type declaration of some reason?